src/base/display.jl

```diff
@@ -1,13 +1,20 @@
-function Base.show(io::IO, drv::Driver)
-    drv.ptr == C_NULL && (return print(io, "NULL Driver"))
+function Base.show(io::IO, drv::Driver)::IO
+    if drv.ptr == C_NULL
+        print(io, "NULL Driver")
+        return io
+    end
     print(io, "Driver: $(shortname(drv))/$(longname(drv))")
+    return io
```

Comment:
Why do we return `io`? Show methods in the language often don't return a thing (`nothing` if you want to be explicit). Same with other `show` methods below. Don't forget about the code annotation if you remove it.

Reply:
> Why do we return `io`? Show methods in the language often don't return a thing 

Thanks! I recall trying to look it up and not getting any guidance on what the convention was (show/print implementations often calling show/print). Done in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/c615a992d155209fd546ba447aab7333c8570374.